assignments/subscriptions/b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb/assign.keyvault_diagnostics.json

```diff
@@ -0,0 +1,36 @@
+{
+    "properties": {
+        "metadata": {
+            "assignedBy": "github.com/hmcts/azure-policy"
+        },
+        "description": "Forwards all KV Logs to an Azure Event Hub",
+        "displayName": "HMCTS Collect and Send KeyVault Logs to Event Hub",
+        "enforcementMode": "Default",
+        "nonComplianceMessages": [],
+        "notScopes": [],
+        "parameters": {
+            "eventHubName": {
```

Comment:
Need a `profileName` parameter here
```
"profileName": {
        "type": "String",
        "metadata": {
          "displayName": "Profile name",
          "description": "The diagnostic settings profile name"
        },
        "defaultValue": "keyVaultToEventHub"
      },
```

Reply:
The default value is already in use, it fails to do anything if we dont have a custom name